app/models/developer.rb

```diff
@@ -23,7 +24,7 @@ class Developer < ApplicationRecord
   validates :bio, presence: true
   validates :time_zone, presence: true, on: :create
   validates :cover_image, content_type: ["image/png", "image/jpeg", "image/jpg"],
-    max_file_size: 10.megabytes
+  max_file_size: 10.megabytes
```

Comment:
Please remove this change - running `bundle exec standardrb --fix` will fix any formatting issues.